### ISSUE TITLE
Test failure: Interop/PInvoke/Miscellaneous/CopyCtor/CopyCtorTest/CopyCtorTest.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#40178]
failed in job: [runtime-coreclr outerloop 20200730.5 ](https://dev.azure.com/dnceng/public/_build/results?buildId=751950&view=ms.vss-test-web.build-test-results-tab&runId=23324156&resultId=100409&paneView=debug)

failed tests:
Interop/PInvoke/Miscellaneous/CopyCtor/CopyCtorTest/CopyCtorTest.sh
Interop\\PInvoke\\Miscellaneous\\CopyCtor\\CopyCtorTest\\CopyCtorTest.cmd

Error message
~~~
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at CopyCtor.StructWithCtorTest(StructWithCtor* ptrStruct, StructWithCtor& refStruct) in /Users/runner/work/1/s/src/tests/Interop/PInvoke/Miscellaneous/CopyCtor/CopyCtorTest.cs:line 15
   at FunctionPointer.Call_FunctionPointer(IntPtr fptr, StructWithCtor* arg1, StructWithCtor& arg2)
   at CopyCtor.Main() in /Users/runner/work/1/s/src/tests/Interop/PInvoke/Miscellaneous/CopyCtor/CopyCtorTest.cs:line 39

Return code:      1
Raw output file:      C:\h\w\A8DA096F\w\B02D0975\e\Interop\PInvoke\Reports\Interop.PInvoke\Miscellaneous\CopyCtor\CopyCtorTest\CopyCtorTest.output.txt
Raw output:
BEGIN EXECUTION
        1 file(s) copied.
" "C:\h\w\A8DA096F\p\crossgen.exe"  /Platform_Assemblies_Paths C:\h\w\A8DA096F\p;C:\h\w\A8DA096F\w\B02D0975\e\Interop\PInvoke\Miscellaneous\CopyCtor\CopyCtorTest /in C:\h\w\A8DA096F\w\B02D0975\e\Interop\PInvoke\Miscellaneous\CopyCtor\CopyCtorTest\CopyCtorTest.org /out C:\h\w\A8DA096F\w\B02D0975\e\Interop\PInvoke\Miscellaneous\CopyCtor\CopyCtorTest\\CopyCtorTest.dll"
Microsoft (R) CoreCLR Native Image Generator - Version 5.0.0-ci
Copyright (c) Microsoft Corporation.  All rights reserved.

Native image C:\h\w\A8DA096F\w\B02D0975\e\Interop\PInvoke\Miscellaneous\CopyCtor\CopyCtorTest\\CopyCtorTest.dll generated successfully.
 "C:\h\w\A8DA096F\p\corerun.exe" CopyCtorTest.dll 
Expected: 100
Actual: -1073741819
END EXECUTION - FAILED
FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=C:\h\w\A8DA096F\p
> C:\h\w\A8DA096F\w\B02D0975\e\Interop\PInvoke\Miscellaneous\CopyCtor\CopyCtorTest\CopyCtorTest.cmd
Expected: True
Actual:   False


Stack trace
   at Interop_PInvoke._Miscellaneous_CopyCtor_CopyCtorTest_CopyCtorTest_._Miscellaneous_CopyCtor_CopyCtorTest_CopyCtorTest_cmd() in F:\workspace\_work\1\s\artifacts\tests\coreclr\Windows_NT.x86.Checked\TestWrappers\Interop.PInvoke\Interop.PInvoke.XUnitWrapper.cs:line 6542
~~~

**Comment by mangod9** [comment]
@davidwrighton since this is a new test which was recently added.

**Comment by davidwrighton** [comment]
This was fixed with prs #40168 and #40161
